src/map/battle.cpp

```diff
@@ -5846,6 +5846,15 @@ static int32 battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list
 				skillratio += (skill_lv * (sstatus->max_sp * 50 / 100) * 5) / 100;
 			}
 
+			RE_LVL_DMOD(100);
+			break;
+		case DK_DRAGONIC_PIERCE:
+			skillratio += -100 + 850 + 600 * skill_lv;
+			skillratio += 7 * sstatus->pow;
```

Comment:
Can you confirm the pow ratio? otherwise :
```suggestion
			skillratio += 7 * sstatus->pow;	// !TODO: unknown ratio
```

Reply:
Nah no information